rails-time-track/db/migrate/20190709220613_create_projects.rb

```diff
@@ -0,0 +1,16 @@
+class CreateProjects < ActiveRecord::Migration[5.2]
+  def change
+    create_table :projects do |t|
+      t.string :name
+      t.string :client
+      t.string :category
+      t.date :start
+      t.date :end
```

Comment:
I think you can't define a attribute with a reserve word like `end`, maybe you can try with `endProjected` or something like that.

Reply:
Thanks, I have modified to end_date and start_date